identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery-otp.jsp

```diff
@@ -125,9 +127,21 @@
     }
 %>
 <%
-    final String USERNAME_CLAIM_URI_FOR_TENANT = "http://wso2.org/claims/username";
+    String user_identifier_claim_key = "http://wso2.org/claims/username";
```

Comment:
This is converted to a variable, since this can now include username and multi attribute identifier key both which will be dynamically assigned. 
Variable name was changed to camel case with https://github.com/wso2/identity-apps/pull/6696/commits/7eaef08ce4f443e5e1c33606aae224f9ff8f1046